babel.config.js

```diff
@@ -1,5 +1,22 @@
+/**
+ *
+ * @param {false | 'auto'} modules
+ * @see https://babeljs.io/docs/en/babel-preset-env#modules
+ */
+const presets = (modules) => [
+  [
+    '@babel/env',
+    {
+      modules,
+    },
+  ],
+  '@babel/react',
+  '@babel/preset-typescript',
+  '@emotion/babel-preset-css-prop',
+];
+
 module.exports = {
-  presets: ['@babel/env', '@babel/react', '@emotion/babel-preset-css-prop'],
+  presets: presets(false),
```

Comment:
note storybook was using modules:'auto' and here is picking up as false.
Also in Chromatic apart from the css class name I'm also seeing `css="[object Object]"` in svg components this is likely related to `babel-preset-css-prop` but I dont see how this is different than old config

Reply:
@daigof The issue was that storybook doesn't automatically pick up `babel.config.js`, only `.babelrc`. So when we removed the `.storybook/.babelrc` it wasn't picking up the other babel config. I've directly imported it in 
af030c4 and now the build is passing.